final_prep/annotations.rb

```diff
@@ -4,40 +4,40 @@
 
 # Build a Bear
 
-def build_a_bear(name, age, fur, clothes, special_power)
-  greeting = "Hey partner! My name is #{name} - will you be my friend?!"
-  demographics = [name, age]
-  power_saying = "Did you know that I can #{special_power}?"
-  built_bear = {
+def build_a_bear(name, age, fur, clothes, special_power) # Creating a method with 3 arguments
+  greeting = "Hey partner! My name is #{name} - will you be my friend?!" # Interpolating first argument into variable called 'greeting'
+  demographics = [name, age] # Creating an array holding our first and second arguments (name and age)
+  power_saying = "Did you know that I can #{special_power}?" # Creating a variable interpolating our 'special_power' argument
+  built_bear = { # Creating a hash using our input arguments
     'basic_info' => demographics,
     'clothes' => clothes,
     'exterior' => fur,
-    'cost' => 49.99,
+    'cost' => 49.99, # Statically declaring 'cost'
     'sayings' => [greeting, power_saying, "Goodnight my friend!"],
-    'is_cuddly' => true,
+    'is_cuddly' => true, # Statically declaring 'is_cuddly'
   }
-  return built_bear
+  return built_bear # Printing our hash to the terminal
```

Comment:
This is actually _returning_ the `built_bear` out of the function (ie. as an output), not printing it to the terminal.